packages/api/bin/serveUtils.js

```diff
@@ -1,72 +1,183 @@
 'use strict';
 
 const indexer = require('@cumulus/es-client/indexer');
+const {
+  GranulesExecutionsPgModel,
+  RulePgModel,
+  GranulePgModel,
+  PdrPgModel,
+  ExecutionPgModel,
+  CollectionPgModel,
+  ProviderPgModel,
+  translateApiCollectionToPostgresCollection,
+  translateApiProviderToPostgresProvider,
+  translateApiExecutionToPostgresExecution,
+  translateApiGranuleToPostgresGranule,
+  translateApiPdrToPostgresPdr,
+  translateApiRuleToPostgresRule,
+  getKnexClient,
+  localStackConnectionEnv,
+  envParams,
+  createTestDatabase,
+  migrationDir,
+} = require('@cumulus/db');
+const { log } = require('console');
 const models = require('../models');
 const { getESClientAndIndex } = require('./local-test-defaults');
+const {
+  erasePostgresTables,
+} = require('./serve');
+
+async function resetPostgresDb() {
+  const knexAdmin = await getKnexClient({ env: localStackConnectionEnv });
+  const knex = await getKnexClient({
+    env: {
+      ...envParams,
+      ...localStackConnectionEnv,
+      migrationDir,
+    },
+  });
+
+  try {
+    await createTestDatabase(knexAdmin, 'postgres', localStackConnectionEnv.PG_USER);
+  } catch (error) {
+    log(`Skipping Postgres DB creation because ${error}`);
+  }
+
+  await knex.migrate.latest();
+
+  await erasePostgresTables(knex);
+}
 
 async function addCollections(collections) {
+  const knex = await getKnexClient({
+    env: {
+      ...envParams,
+      ...localStackConnectionEnv,
+    },
+  });
+
   const collectionModel = new models.Collection();
   const es = await getESClientAndIndex();
   return await Promise.all(
-    collections.map((c) =>
-      collectionModel
-        .create(c)
-        .then((collection) =>
-          indexer.indexCollection(es.client, collection, es.index)))
+    collections.map(async (c) => {
+      const dynamoRecord = await collectionModel.create(c);
+      await indexer.indexCollection(es.client, dynamoRecord, es.index);
+      const dbRecord = await translateApiCollectionToPostgresCollection(c);
+      const collectionPgModel = new CollectionPgModel();
+      await collectionPgModel.create(knex, dbRecord);
+    })
   );
```

Comment:
Updated each `addXXX` method to create a postgres record using the appropriate `translateApiXxxToPostgresXxx()` and then calling `xxxPgModel.create()`